src/mono/wasm/testassets/WasmBasicTestApp/LazyLoadingTest.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using System.Text.Json;
+using System.Runtime.InteropServices.JavaScript;
+
+public partial class LazyLoadingTest
+{
+    [JSExport]
```

Comment:
Could you please add comment explaining what make this lazy ?

Reply:
Like this?

> System.Text.Json is marked as lazy loaded in the csproj, this method can be called only after the assembly is lazy loaded
> In the test case it is done in the JS before call to this method

Reply:
And also the name of the msbuild property which does it